src/Passes/.clang-tidy

```diff
@@ -0,0 +1,76 @@
+Checks: "-*,bugprone-*,\
+-readability-*,\
+readability-identifier-*,\
+readability-braces-around-statements,\
+cert-dcl*,\
+cert-env*,\
+cert-err52-cpp,\
+cert-err60-cpp,\
+cert-flp30-c,\
+clang-analyzer-security.FloatLoopCounter,\
+google-build-explicit-make-pair,\
+google-build-namespaces,\
+google-explicit-constructor,\
+google-readability-*,\
+google-runtime-operator,\
+hicpp-exception-baseclass,\
+hicpp-explicit-conversions,\
+hicpp-use-*,\
+misc-*,\
+-misc-misplaced-widening-cast,\
+performance-*"
+
+WarningsAsErrors: '*'
+HeaderFilterRegex: '.*'
+
+CheckOptions:
+  - key:    readability-identifier-naming.ClassCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.AbstractClassPrefix
+    value:  'I'
+  - key:    readability-identifier-naming.StructCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.ParameterCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.PrivateMemberCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.PrivateMemberSuffix
+    value:  '_'    
+  - key:    readability-identifier-naming.ProtectedMemberCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.ProtectedMemberSuffix
+    value:  '_'        
+  - key:    readability-identifier-naming.VariableCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.LocalVariableCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.TypeAliasCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.UnionCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.FunctionCase
+    value:  'camelBack'
+  - key:    readability-identifier-naming.NamespaceCase
+    value:  'lower_case'
+  - key:    readability-identifier-naming.GlobalConstantCase
+    value:  'UPPER_CASE'
+  - key:    readability-identifier-naming.EnumCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.EnumConstantCase
+    value:  'CamelCase'
+  - key:    readability-identifier-naming.GlobalConstantPrefix
+    value:  'G_'
+  - key:    readability-identifier-naming.ConstantCase
+    value: 'UPPER_CASE'
+  - key:    readability-identifier-naming.MacroDefinitionCase
+    value:  'UPPER_CASE'
+  - key:    readability-identifier-naming.TypeAliasCase
+    value:  'CamelCase'
```

Comment:
FYI: I see `readability-identifier-naming.TypeAliasCase` twice. You likely meant once.

Reply:
Yes, this is a mistake. I've remove the second entry.